src/components/SecondaryNav/SecondaryNav.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+/**
+ * Secondary navigation typically used under the header
+ *
+ * Must pass an array of NavLinks for the navigation bar.
+ * Each NavLink should contain an anchor with appropriate href, or a react
+ * router Link or equivelant.
+ */
+const SecondaryNav = ({ buttons, className }) => {
```

Comment:
would like to see some `aria` labels and a11y features added into this before releasing.

This is a good resource to understand the aria lable for tabs, we can make another PR to address the js part of it later
https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html